README.md

```diff
@@ -518,6 +519,42 @@ Since the output of FHIR expression is flexible, users should provide expression
 1. We support FHIR data in R4 and STU 3, JSON format. Support for XML is planned.
 2. De-identification of fields within Extensions is not supported. 
 
+## Using Presidio to de-identify PII in free text fields
+[Presidio](https://github.com/Microsoft/presidio) helps to ensure sensitive data is properly managed and governed. It provides fast identification and anonymization modules for private entities in text such as credit card numbers, names, locations, social security numbers, bitcoin wallets, US phone numbers, financial data and more.
+
+In cases of free text in the FHIR file format, it's possible to anonymize only sensitive data, leaving the rest of the text in place.
+### Pre-requisites
+1. [Docker](https://docs.docker.com/get-docker/)
+
+### Follow the instructions to enable Presidio:
+1. Download Presidio's docker-compose file and execute it:
+```
+curl https://raw.githubusercontent.com/microsoft/presidio/main/docker-compose.yml --output docker-compose.yml

```

Comment:
I'm not sure that the compose file works on its own, did you try?
It can download the containers from mcr, but REGISTRY_NAME might need to be overriden